sqlmesh/dbt/manifest.py

```diff
@@ -41,7 +41,7 @@
 MacroConfigs = t.Dict[str, MacroConfig]
 
 
-BUILTIN_CALLS = {*BUILTIN_GLOBALS, *BUILTIN_FILTERS}
+BUILTIN_CALLS = {*BUILTIN_GLOBALS, *BUILTIN_FILTERS, "is_incremental"}
```

Comment:
There are other bulitin calls besides `is_incremental` that are added in `create_builtin_globals` -- `flags` as one example. Is there any impact in omitting them here?

Reply:
That's an artifact that I forgot to remove. This should not be here. Thanks for catching.